sql/driver/driver_test.go

```diff
@@ -288,3 +290,125 @@ func TestInsecure(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func strToInts(t *testing.T, results resultSlice) []int {
```

Comment:
This method is mainly undoing what `readAll` did; maybe we should have a `readAllInts` function that converts `Rows` directly into a `[]int`.